Cannot build on Antegros Linux

```
  = note: "cc" "-Wl,--as-needed" "-Wl,-z,noexecstack" "-m64" "-L" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.17avbr6m4tgnebgh.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.1i0dpxzd6urazmyy.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.1ns3gnuctqdjv9ko.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.1s6icq4dbkl3n09e.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.30yv88pup3n8a4ko.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.37jmeyn16oirazjd.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.3mrn8atmgzehavte.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.4g81oxqoermpirv2.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.4zffp4tb2v7wddd3.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.afxdj38tqyq35rt.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.hehwu40x2vn61cs.rcgu.o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.t9bzsbcthapoxoe.rcgu.o" "-o" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d" "/home/eralp/Codes/Rust/test2/target/debug/deps/test2-edffb4c11d52145d.asa1woamuvxwnm0.rcgu.o" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-nodefaultlibs" "-L" "/home/eralp/Codes/Rust/test2/target/debug/deps" "-L" "/home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out" "-L" "/usr/lib" "-L" "/home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out" "-L" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/home/eralp/Codes/Rust/test2/target/debug/deps/libtcod-64be57a909571c22.rlib" "/home/eralp/Codes/Rust/test2/target/debug/deps/libtcod_sys-2047bdb8c559a91e.rlib" "/home/eralp/Codes/Rust/test2/target/debug/deps/liblazy_static-413acabfe4abac7f.rlib" "/home/eralp/Codes/Rust/test2/target/debug/deps/libbitflags-9d93ebba08c806ab.rlib" "-Wl,--start-group" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-e0b6e478e9762da9.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-765c392663bd34b7.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libbacktrace_sys-1059e0ba7f05fd67.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-85b43da92537c77f.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-f8a77019eff82b98.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-1d32a47a0bdcb0a2.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-c47e11e0b4c869e2.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-5a60e280b382f06e.rlib" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-8010f7064010be9c.rlib" "-Wl,--end-group" "/home/eralp/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-566972fa63f867ee.rlib" "-Wl,-Bdynamic" "-lSDL2" "-ltcod" "-ldl" "-lrt" "-lpthread" "-lgcc_s" "-lc" "-lm" "-lrt" "-lpthread" "-lutil" "-lutil"
  = note: /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_c.o): in function `TCOD_sys_load_font':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:204: undefined reference to `SDL_FreeSurface'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:235: undefined reference to `SDL_UpperBlit'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:236: undefined reference to `SDL_FreeSurface'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:266: undefined reference to `SDL_UpperBlit'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:267: undefined reference to `SDL_FreeSurface'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:305: undefined reference to `SDL_UpperBlit'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:306: undefined reference to `SDL_FreeSurface'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:338: undefined reference to `SDL_MapRGB'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:342: undefined reference to `SDL_SetColorKey'
          /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_c.o): in function `TCOD_sys_console_to_bitmap':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:437: undefined reference to `SDL_UpperBlit'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:467: undefined reference to `SDL_MapRGB'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:468: undefined reference to `SDL_FillRect'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:496: undefined reference to `SDL_MapRGB'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:593: undefined reference to `SDL_UpperBlit'
          /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_c.o): in function `TCOD_sys_startup':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:660: undefined reference to `SDL_Init'
          /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_c.o): in function `TCOD_sys_shutdown':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:675: undefined reference to `SDL_Quit'
          /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_c.o): in function `TCOD_sys_load_player_config':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:726: undefined reference to `SDL_FreeSurface'
          /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_c.o): in function `TCOD_sys_delete_bitmap':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:1559: undefined reference to `SDL_FreeSurface'
          /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_c.o): in function `TCOD_sys_check_magic_number':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_c.c:1598: undefined reference to `SDL_RWFromFile'
          /usr/bin/ld: /home/eralp/Codes/Rust/test2/target/debug/build/tcod-sys-86f38c8095f34fd1/out/libtcod.a(sys_sdl_img_bmp.o): in function `TCOD_sys_read_bmp':
          /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_img_bmp.c:41: undefined reference to `SDL_RWFromFile'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_img_bmp.c:41: undefined reference to `SDL_LoadBMP_RW'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_img_bmp.c:42: undefined reference to `SDL_GetError'
          /usr/bin/ld: /home/eralp/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-5.0.0/libtcod/src/sys_sdl_img_bmp.c:56: undefined reference to `SDL_CreateRGBSurface'
```

This goes on. It cannot link some object files even tho I have them installed ( even SDL Libraries)  Considering the output there might be an issue with the installation of the library but I am not sure.

--- a/tcod_sys/build.rs
+++ b/tcod_sys/build.rs
@@ -312,4 +312,5 @@ fn main() {
 
     println!("cargo:rustc-link-search={}", dst.display());
     println!("cargo:rustc-link-lib=dylib=tcod");
+    println!("cargo:rustc-link-lib=SDL2");
 }

